packages/contracts/hardhat.config.ts

```diff
@@ -1,8 +1,25 @@
 import { HardhatUserConfig } from 'hardhat/config'
 import '@nomicfoundation/hardhat-toolbox'
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+require('dotenv').config()
 
 const config: HardhatUserConfig = {
+  networks: {
+    hardhat: {
+      forking: {
+        url: `https://mainnet.infura.io/v3/${process.env.PROVIDER_KEY}`,
```

Comment:
I don't think we should use main net here

Reply:
Can you provide more information? And what should be instead of `mainnet`?

This is a specification for `hardhat` networks, this is used to run tests from a `mainnet` fork that starts from the block `7327025`, to be able to emulate the conditions for the proof generated, with the test QR code with the timestamp `20190308114407437`, to be validated. This does not impact the deployment flow, and it's only a setting for hardhat while running the test. 
Also, Sepolia could not be set here, as it does not have blocks minted before 2021.

Reply:
Oh, Got it! Thank you explanation!